packages/asc-ui/src/components/Pagination/PaginationStyle.ts

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { Props as CompactPagerProps } from '../CompactPager/CompactPager'
+import { svgFill, themeColor } from '../../utils'
+import Button from '../Button'
+
+export type Props = CompactPagerProps
+
+export default styled.nav<Props>`
+  ul {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    list-style: none;
+  }
+
+  li {
+    display: list-item;
+    margin: 0 15px 0 0;
```

Comment:
I think we should use 16px here. Design system is made with 5px, but we made an agreement that components should be build on a unit of 4px. Therefore we could use themeSpacing(4) here

Reply:
Fixed

Reply:
Fixed